rebasehelper/helpers/bugzilla_helper.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you rebase your package to the latest version
+# Copyright (C) 2013-2019 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hráček <phracek@redhat.com>
+#          Tomáš Hozza <thozza@redhat.com>
+#          Nikola Forró <nforro@redhat.com>
+#          František Nečas <fifinecas@seznam.cz>
+
+import logging
+import os
+import re
+from typing import cast, Optional, Tuple
+
+import git  # type: ignore
+import requests
+
+from rebasehelper.exceptions import RebaseHelperError
+from rebasehelper.logger import CustomLogger
+
+
+logger: CustomLogger = cast(CustomLogger, logging.getLogger(__name__))
+
+
+class BugzillaHelper:
+
+    """Class for working with Upstream Release Monitoring on bugzilla."""
+
+    REPO_URL = 'https://src.fedoraproject.org/rpms/{}.git'
+    BUGZILLA_REST = 'https://bugzilla.redhat.com/rest/bug'
+
+    @classmethod
+    def get_bugzilla_component(cls, bugzilla_id: str) -> str:
+        """Gets a component of the bugzilla.
+
+        Args:
+            bugzilla_id: ID of the bugzilla.
+
+        Returns:
+            Component of the bugzilla.
+
+        Raises:
+            RebaseHelperError: If no such bugzilla exists or if the bugzilla
+            is not created by Upstream Release Monitoring.
+
+        """
+        r = requests.get('{}/{}'.format(cls.BUGZILLA_REST, bugzilla_id))
+        if not r.ok:
+            raise RebaseHelperError('Could not obtain data from bugzilla')
+
+        if 'error' in r.json():
+            raise RebaseHelperError('No bugzilla with ID {} was found'.format(bugzilla_id))
+
+        bug = r.json()['bugs'][0]
```

Comment:
A nitpick, but I would avoid using `r.json()` twice, the response data have to be parsed twice as well.